hack/generator/pkg/codegen/testdata/ArmResource/Arm_test_simple_resource_array_properties_azure.golden

```diff
@@ -335,31 +335,31 @@ func (fakeResourceSpec *FakeResource_Spec) PopulateFromARM(owner genruntime.Know
 	// Set property ‘ArrayOfArrays’:
 	for _, item := range typedInput.ArrayOfArrays {
 		var itemTemp []Foo
-		for _, item1 := range item {
-			var item2 Foo
-			err := item2.PopulateFromARM(owner, item1)
+		for _, itemItem := range item {
+			var itemItem1 Foo
+			err := itemItem1.PopulateFromARM(owner, itemItem)
 			if err != nil {
 				return err
 			}
-			itemTemp = append(itemTemp, item2)
+			itemTemp = append(itemTemp, itemItem1)
 		}
 		fakeResourceSpec.ArrayOfArrays = append(fakeResourceSpec.ArrayOfArrays, itemTemp)
 	}
 
 	// Set property ‘ArrayOfArraysOfArrays’:
 	for _, item := range typedInput.ArrayOfArraysOfArrays {
 		var itemTemp [][]Foo
-		for _, item1 := range item {
-			var item1Temp []Foo
-			for _, item2 := range item1 {
-				var item3 Foo
-				err := item3.PopulateFromARM(owner, item2)
+		for _, itemItem := range item {
+			var itemItemTemp []Foo
+			for _, itemItemItem := range itemItem {
+				var itemItemItem1 Foo
```

Comment:
These ones haven't come out very well - any idea how to make it better @theunrepentantgeek?